cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -248,21 +292,26 @@
     "note": {
       "properties": {
         "description": {
+          "description": "Any other notes that need to be displayed prominently in the record",
```

Comment:
"displayed" is a secondary motivation here, better say something like content-related, e.g. just "Notes of interest for the given resource" or something of the kind